backend/pick-git/src/main/java/com/woowacourse/pickgit/post/infrastructure/extractor/GithubRepositorySearchExtractor.java

```diff
@@ -52,9 +52,10 @@ private String generateApiUrl(
         int page,
         int limit
     ) {
-
+        String format = apiBaseUrl +
+            "/search/repositories?q=user:%s %s in:name fork:true&page=%d&per_page=%d";
         return String.format(
```

Comment:
```suggestion
        return String.format(format, username, keyword, page, limit);
```

코드 길이가 짧으니까 1줄로 적어도 될 것 같아요!